GOVERNANCE.md

```diff
@@ -0,0 +1,66 @@
+### Documentation Working Group
+
+The Node.js Documentation group is jointly governed by a Working Group (WG) which is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+README.md.
+
+### Collaborators
```

Comment:
Yes, I saw that the contributor additions were also noted in CONTRIBUTING.md

This is the only part that really seems to fully overlap is this:

>To join the team:
>
>If you'd like to join the documentation team, that's great! Note your interest on this issue. We'll get in contact with you shortly afterwards. Note that as part of the team, you are expected to be an exemplar of the project's values – both in making this specific project a welcoming place to contribute, as well as in the larger community (other issue trackers, conferences, etc.)
>
>Every month, new members will be added to the team. If you are added you will be paired with an existing editor from the team who will be responsible for your work for one month, upon successful completion of which you will become a full member.

Which end should be modified?